nebula_ros/include/nebula_ros/continental/continental_ars548_hw_interface_wrapper.hpp

```diff
@@ -66,7 +66,7 @@ class ContinentalARS548HwInterfaceWrapper
 
   /// @brief Callback to send the steering angle information to the radar device
   /// @param msg The steering angle message
-  void SteeringAngleCallback(const std_msgs::msg::Float32::SharedPtr msg);
+  void steeringAngleCallback(const std_msgs::msg::Float32::SharedPtr msg);
```

Comment:
I checked manually and the code compiles even without this change.
Plus, all the other are with CamelCase

Reply:
Addressed in [7abe20f](https://github.com/tier4/nebula/pull/187/commits/7abe20f7fd84b2dbd052cc0272febb6946c3b5c3) :+1: